metamorph/store/dynamodb/dynamodb.go

```diff
@@ -43,15 +43,21 @@ func New() (store.MetamorphStore, error) {
 	if err != nil {
 		return dynamodbClient, err
 	} else if !exists {
```

Comment:
I think this if can be flattened this way:
```
if err != nil {
  return dynamo ...
]

if !exists {

}
``` without else. Looks slightly more readable to me.